nuxeo-common/src/test/java/org/nuxeo/common/xmap/Author.java

```diff
@@ -197,4 +197,7 @@ public String getField3() {
 
     }
 
+    @XNode("@class")
+    Class<?> klass;
```

Comment:
This is for `XMapTest.testInvalidClass` in `nuxeo-common`. `DummyContribution` is part of `nuxeo-runtime-test`.